app/assets/stylesheets/application/medications.scss

```diff
@@ -9,6 +9,25 @@
 	height: 150px;
 }
 
+#medication_content {
+  padding: 2.5em 10em;
```

Comment:
Hi Julia! I used different values for padding on this specific page as without them the larger-width page looks like this (unless you meant something else when saying "use same values for padding/margin," let me know if I read that wrong):

<img width="1436" alt="screen shot 2016-10-12 at 10 32 48 pm" src="https://cloud.githubusercontent.com/assets/13127241/19335828/f7b18474-90cb-11e6-9bb5-02b0c299074b.png">

Comparing the above screenshot with the one that includes more padding for this page, I agree with what @aayushpi said earlier this year about how utilizing the entire screen width for content display isn't always the best option ("IMO no, there’s a lot of research about fields that are longer than they typically need to be being left incomplete. Our data set is small, so the fields should be as such"). 

Whitespace can be a benefit to effectively utilizing the "real estate" of the page for optimal user experience (from what I've learned so far, at least; I welcome others' opinions, though!), especially on larger screens. I was basically aiming for better readability, to feel more like filling out a typical form than the original, and something that's not too crowded looking or too isolated. 

If it's an absolute must to keep to the same padding/margin on this page, I can work on creating a different design in Sketch and start over (or left-align everything? though that would probably leave _too_ much whitespace on the right side, it's always a balancing act with that... [thinking to myself here, but I experimented anyways and got the following:

<img width="1432" alt="screen shot 2016-10-12 at 11 05 30 pm" src="https://cloud.githubusercontent.com/assets/13127241/19336382/798d6c70-90d0-11e6-8d75-3e6b5981f4eb.png">
]). 

Oh, and I can definitely convert the ems to pxs for this project, of course. 👍 😀